doc/basics.md

```diff
@@ -1016,6 +1017,102 @@ If your documents are large (e.g., larger than a megabyte), then the `iterate_ma
 
 See [iterate_many.md](iterate_many.md) for detailed information and design.
 
+
+
+Parsing Numbers Inside Strings
+------------------------------
+
+The simdjson library supports parsing valid numbers inside strings through three methods: `get_double_in_string`, `get_int64_in_string` and  `get_uint64_in_string`. However, it is important to
+note that these methods are not substitute to the regular `get_double`, `get_int64` and `get_uint64`. The usage of the `get_*_in_string` methods is solely to parse numbers inside strings, and so
```

Comment:
I would add here or elsewhere something like this...

"The number must be valid JSON numbers with no leading or trailing space. In particular, the strings `"nan"`, `"1e"` or `"infinity"` will not be accepted as valid numbers."